client/src/deploy/creation_common.rs

```diff
@@ -939,3 +940,23 @@ pub(super) mod payment_version {
         matches.value_of(ARG_NAME).unwrap_or_default()
     }
 }
+
+pub(super) mod force {
```

Comment:
We shouldn't need this module at all - there's a suitable one in common.rs.

[Its `arg()` function](https://github.com/casper-network/casper-node/blob/dev/client/src/common.rs#L128-L139) can be passed the `DisplayOrder::Force` which you added above, and in each case the `singular` arg should be true.